pages/news/2022/10-26-fpm-version-0.7.0.md

```diff
@@ -0,0 +1,51 @@
+---
+author: Lewis McMillan, Damian Rouson, Giannis Nikiteas, Laurence Kedward, Sebastian Ehlert, Zuo Zhihua, Jakub Jelínek, Arteev Raina
+date: 2022-10-26
+category: release
+...
+
+# Fpm version 0.7.0 released
+
+This release introduces preprocessor support in the `fpm.toml` manifest, allowing
+for preprocessor macros, file suffixes and directories where the preprocessor should run.
```

Comment:
Thank you for drafting this release note. I think it's in great shape, but only suggest acknowledging GSoC for the preproc table feature:

```suggestion
for preprocessor macros, file suffixes and directories where the preprocessor should run.
This feature was implemented as part of Arteev Raina's Google Summer of Code 2022 project.
```